lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -383,6 +383,97 @@ describe('FilePickerApp', () => {
     assert.isFalse(wrapper.exists('ErrorModal'));
   });
 
+  context('when editing an existing assignment', () => {
+    beforeEach(() => {
+      fakeConfig.assignment = {
+        document: {
+          url: 'https://test.com/example.pdf',
+        },
+      };
+
+      // FIXME - The existing assignment configuration and "Save" buttons
+      // should be visible even if groups are not enabled.
+      fakeConfig.product.settings.groupsEnabled = true;
+    });
+
+    /** Click the button to edit the existing content selection. */
+    function clickEditButton(wrapper) {
+      const editButton = wrapper.find('Button[data-testid="edit-content"]');
+      assert.isTrue(editButton.exists());
+      act(() => {
+        editButton.prop('onClick')();
+      });
+      wrapper.update();
+    }
+
+    /** Click the button to cancel editing the content selection. */
+    function clickCancelEditButton(wrapper) {
+      const cancelButton = wrapper.find(
+        'Button[data-testid="cancel-edit-content"]'
+      );
+      assert.isTrue(cancelButton.exists());
+      act(() => {
+        cancelButton.prop('onClick')();
+      });
+      wrapper.update();
+    }
```

Comment:
I have replaced these two functions with one that accepts a `data-testid` value and searches for a matching `button[data-testid=${value}]`, on which it uses `simulate('click')`. This works since third-party components are not mocked by default and `data-testid` is forwarded to the underlying component.